tests/apps/courses/test_cms_plugins_course.py

```diff
@@ -20,6 +20,31 @@ class CoursePluginTestCase(TestCase):
     Test that CoursePlugin correctly displays a Course's page placeholders content
     """
 
+    @staticmethod
+    def test_cms_plugins_course_unique():
```

Comment:
I think it would be clearer to remove this test and modify the above test. You just need to replace:
```python
self.assertIn(course.extended_object.get_title(), plugin_form.as_table())
```
by 
```python
rendered_form = plugin_form.as_table()
self.assertEqual(rendered_form.count(course.extended_object.get_title()), 1)
```

and update the docstring...

Reply:
done